packages/js-core/src/website/lib/sync.ts

```diff
@@ -68,6 +68,7 @@ export const sync = async (params: TJsWebsiteSyncParams, noCache = false): Promi
     let state: TJsWebsiteState = {
       surveys: syncResult.value.surveys as TSurvey[],
       noCodeActionClasses: syncResult.value.noCodeActionClasses,
+      ...(syncResult.value.actionClasses && { actionClasses: syncResult.value.actionClasses }),
```

Comment:
why the special syntax? Don't we always get `actionClasses` from the sync endpoint (in this version)?